File: expressions/budget.py
income=float(input("Hello, and welcome to your financial calculator!\n What is your monthly income: "))
rent=float(input("What is your monthly rent: "))
utilities=float(input("What is your monthly utilities: "))
groceries=float(input("What is your monthly groceries: "))
transportation=float(input("What is your monthly transportation costs: "))
saving = income*2
expenses = rent+ utilities + groceries + transportation
spend = income - expenses - saving
print(f"Your monthly income is ${income:.2f}\n")
print(f"Your monthly expenses are ${expenses:.2f}\n")
print(f"Your monthly savings is ${saving:.2f}\n")
print(f"Your monthly spending money is ${spend:.2f}\n")
prent = rent/ income *100
putilities = utilities/income *100
pgroceries = groceries/income *100
ptransportation = transportation/income * 100
psaving = saving/income *100
pexpenses = expenses/income *100
print(f"Your rent is {int(prent)}% of your monthly income\n")
print(f"Your utilities are {int(putilities)}% of your monthly income\n")
print(f"Your groceries are {int(pgroceries)}% of your monthly income\n")
print(f"Your transportation is {int(ptransportation)}% of your monthly income\n")
print(f"Your savings are {int(psaving)}% of your monthly income\n")
print(f"Your expenses are {int(pexpenses)}% of your monthly income\n")

File: expressions/budget.c
#include <stdio.h>

int main(void){
    float income, rent, utilities, grocereis, transportation, savings, expenses, spend;
    float prent, putilities, pgroceries, ptransportation, psaving, pexpenses;
    printf("This is a budget calculator,\n How much do you make a month?\n");
    scanf("%f", &income);
    printf("How much does your rent cost?\n");
    scanf("%f", &rent);
    printf("How much does your utilites cost?\n");
    scanf("%f", &utilities);
    printf("How much do your grocereies cost?\n");
    scanf("%f", &grocereis);
    printf("How much does your transportation cost?\n");
    scanf("%f", &transportation);
    savings = income * .2;
    expenses = rent+ utilities + grocereis + transportation;
    spend = income - expenses - savings;
    printf("You make %.2f\n", income);
    printf("You savings are  $%.2f\n", savings);
    printf("You expenses are $%.2f\n", expenses); 
    printf("You spending money is $%.2f\n", spend);
    prent = rent/ income *100;
    putilities = utilities/income *100;
    pgroceries = grocereis/income *100;
    ptransportation = transportation/income * 100;
    psaving = savings/income *100;
    pexpenses = expenses/income *100;
    printf("Your rent is %d%% of your income.\n", (int) prent);
    printf("Your utilities is %d%% of your income.\n", (int) putilities);
    printf("Your groceries is %d%% of your income.\n", (int) pgroceries);
    printf("Your transportation is %d%% of your income.\n", (int) ptransportation);
    printf("Your savings is %d%% of your income.\n", (int) psaving);
return 0;
}